scripts/checker.sh

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env bash
+
+readonly RED='\033[0;31m'
+readonly GREEN='\033[0;32m'
+readonly INVERTED='\033[7m'
+readonly NC='\033[0m' # No Color
+
+# DEFAULTS
+helmLint=false
+directories=()
+helmVersion=latest
+exitImmediatelyIfErr="set -e"
+doNotExitImmediatelyIfErr="set +e"
+
+function show_help() {
+    echo " Bundle checking tool"
+    echo " Usage:"
+    echo "   checker.sh [flags]"
+    echo
+    echo " Flags:"
+    echo "    -h --help         helm for the script"
+    echo "    --helm-lint       set to perform helm lint operation"
+    echo "    --directories     specifies the list of bundle repository separated by ;"
+    echo "    --helm-version    specifies the Helm version. Default set to `latest`."
+    echo
+    echo " Example of usage:"
+    echo " checker.sh --helm-lint --directories ./bundles/showcase;./bundles/stable --helm-version v2.10.0"
+}
+
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+    key="$1"
+
+    case ${key} in
+        -h|-\?|--help)
+            show_help
+            exit 0
+            ;;
+        --helm-lint)
+            helmLint=true
+            shift
+            ;;
+        --directories)
+            directories=(${2//;/ })
+            shift
+            shift
+            ;;
+        --helm-version)
+            helmVersion="$2"
+            shift
+            shift
+            ;;
+        *)    # unknown option
+            POSITIONAL+=("$1") # save it in an array for later
+            shift # past argument
+            ;;
+    esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+function executeCmd() {
+    local cmd=$1
+    local exitErr=0
+
+    echo -e "\n${INVERTED}Executing: ${cmd}${NC}"
+    local OUT_COLOR=${GREEN}
+
+    # To prevent getting lost of new line characters, see: https://unix.stackexchange.com/a/164548
+    IFS=
+    out=$(eval "${cmd}")
+    if [ $? -eq 1 ];
+    then
+        OUT_COLOR=${RED}
+        exitErr=1
+    fi
+
+    echo -e "${OUT_COLOR}"
+    echo ${out}
+    echo -e "${NC}"
+
+    return ${exitErr}
+}
+
+function downloadChecker() {
+    echo "Installing bundle checker"
+    go get "github.com/kyma-project/kyma/components/helm-broker/cmd/checker"
```

Comment:
I am afraid about one situation: if you already have helm-broker on your local machine, this command will not update it. So later you can run checker with a very old version. On CI it will probably use the most up-to-date version but this can be super confusing for ppl to find out why locally everything is fine and then on CI it is failing (because checker added new rules)

Reply:
**totally agree with you**, because of that I've added it to the follow-up issue to change the way how we are downloading the tools binaries. For now it will be "partially" solved by adding `-u` flag